onnx/defs/math/defs.cc

```diff
@@ -3427,72 +3493,63 @@ static const char* DFT_ver16_doc =
                 "If onesided is 1, only values for w in [0, 1, 2, ..., floor(n_fft/2) + 1] are returned because "
                 "the real-to-complex Fourier transform satisfies the conjugate symmetry, i.e., X[m, w] = X[m,w]=X[m,n_fft-w]*. "
                 "Note if the input or window tensors are complex, then onesided output is not possible. "
-                "Enabling onesided with real inputs performs a Real-valued fast Fourier transform (RFFT)."
+                "Enabling onesided with real inputs performs a Real-valued fast Fourier transform (RFFT). "
                 "When invoked with real or complex valued input, the default value is 0. "
                 "Values can be 0 or 1.",
                 AttributeProto::INT,
                 static_cast<int64_t>(0))
             .Attr(
                 "axis",
-                "The axis on which to perform the DFT. By default this value is set to 0, which corresponds to the first dimension after the batch index."
-                "This value must be less than signal_dimN, where signal_dimN is the number of dimensions in the signal.",
+                "The axis on which to perform the DFT. By default this value is set to 1, which corresponds to the first dimension after the batch index.",
                 AttributeProto::INT,
-                static_cast<int64_t>(0))
+                static_cast<int64_t>(1))
             .Input(
                 0,
                 "input",
-                "For real input, the following shape is expected: [batch_idx][n_fft]."
-                "For complex input, the following shape is expected: [batch_idx][n_fft][2]." 
-                "The final dimension represents the real and imaginary parts of the value."
-                "For real multi-dimensional input, the following shape is expected: [batch_idx][signal_dim1][signal_dim2]...[signal_dimN][1]."
-                "For complex multi-dimensional input, the following shape is expected: [batch_idx][signal_dim1][signal_dim2]...[signal_dimN][2]."
-                "The first dimension is the batch dimension.",
-                "T",
+                "For real input, the following shape is expected: [batch_idx][signal_dim1][signal_dim2]...[signal_dimN][1]. "
+                "For complex input, the following shape is expected: [batch_idx][signal_dim1][signal_dim2]...[signal_dimN][2]. "
+                "The first dimension is the batch dimension. "
+                "The following N dimentions correspond to the signal's dimensions. "
+                "The final dimension represents the real and imaginary parts of the value.",
```

Comment:
Please explicitly specify the order. Eg., if real comes first, then I suggest adding the words `in that order`